src/main/scripts/preinstall.sh

```diff
@@ -1,8 +1,30 @@
 #!/usr/bin/env bash
 
-if id "julie-kafka" &>/dev/null;
-then
-  echo "User julie-kafka already exist"
-else
-   adduser -M -r julie-kafka
-fi
+function default_user_creation_if_required {
+  if id "julie-kafka" &>/dev/null;
+  then
+    echo "User julie-kafka already exist"
+  else
+    adduser -M -r julie-kafka
+  fi
+}
+
+function verify_java_version_min_11 {
+    current_version=$(java -version 2>&1 | tr "\n" " " | cut -d " " -f 3  | tr -d '"' | cut -d "." -f 1)
+    if (( $current_version < 11 ))
```

Comment:
```suggestion
    if (( current_version < 11 ))
```

Reply:
Imho the  `$` is not necessary here

Reply:
I guess old habits die hard, good catch man